app/src/main/java/com/keylesspalace/tusky/components/systemnotifications/NotificationService.kt

```diff
@@ -583,16 +581,11 @@ class NotificationService @Inject constructor(
     }
 
     private fun joinNames(notifications1: List<StatusBarNotification>, notifications2: List<Notification>): String? {
-        val names = java.util.ArrayList<String>(notifications1.size + notifications2.size)
+        val names1 = notifications1.map { it.notification.extras.getString(EXTRA_ACCOUNT_NAME) }
```

Comment:
Use `mapNotNull` here and save the `filterNotNull()` below

Reply:
Done.